### DISCUSSION TITLE
Any Insights on NVCC and Clang Difference In Generated Code Performance?

### DISCUSSION THREAD

**Discussion by K-Wu** [NVIDIA/cutlass#935] [General]
I am interested in using cutlass profiler as a benchmark to see the performance difference of generated code by nvcc and clang. I noticed that cutlass dropped the support for clang since 3.0. While I am going to use cutlass 2 to do the experiment, may I know if anyone has any intuition of experience with the code performance by nvcc and clang? Many thanks.

**Comment by hwu36** [comment]
We do not officially support clang even during 2.x time, but we tested with clang with 2.x from time to time and fixed all the building issues we found in 2.x.

In 3.x time, we only fully supports nvcc for now.  We are working on supporting other compilers.  We welcome the community to help us on this effort.
